src/main/java/org/tron/common/backup/BackupServer.java

```diff
@@ -0,0 +1,103 @@
+package org.tron.common.backup;
+
+import io.netty.bootstrap.Bootstrap;
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.nio.NioEventLoopGroup;
+import io.netty.channel.socket.nio.NioDatagramChannel;
+import io.netty.handler.codec.protobuf.ProtobufVarint32FrameDecoder;
+import io.netty.handler.codec.protobuf.ProtobufVarint32LengthFieldPrepender;
+import java.net.BindException;
+import java.util.concurrent.TimeUnit;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.tron.common.overlay.discover.DiscoveryExecutor;
+import org.tron.common.net.udp.handler.MessageHandler;
+import org.tron.common.net.udp.handler.PacketDecoder;
+import org.tron.common.overlay.server.WireTrafficStats;
+import org.tron.core.config.args.Args;
+
+@Component
+public class BackupServer {
+
+  private static final org.slf4j.Logger logger = LoggerFactory.getLogger("BackupServer");
+
+  private Args args = Args.getInstance();
+
+  private int port = args.getBackupPort();
+
+  private BackupManager backupManager;
+
+  private Channel channel;
+
+  private volatile boolean shutdown = false;
+
+  @Autowired
+  WireTrafficStats stats;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Explicitly declare the visibility for "stats". ([squid:S2039](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2039))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=951&issues=AWPTJmxfA0k93xG_IQRw&open=AWPTJmxfA0k93xG_IQRw)

[ISSUE_KEY]: AWPTJmxfA0k93xG_IQRw